migration/migrating_3_4/migrating-applications-with-cam.adoc

```diff
@@ -6,7 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/migration-launching-cam.adoc[leveloffset=+1]
+You can migrate application workloads by adding your clusters and replication repository to the CAM web console. Then you can create and run a migration plan.
```

Comment:
I think this sound better as one sentence; something like: 

"You can migrate application workloads by adding your clusters and replication repository to the CAM web console to create and run a migration plan."

Reply:
No, that changes the meaning too much. Each one is a separate step:

- Add clusters
- Add repo
- Create migration plan
- Run migration plan

Reply:
Not changing.